pkg/pillar/cmd/volumemgr/handlecontent.go

```diff
@@ -74,10 +59,11 @@ func handleContentTreeDelete(ctxArg interface{}, key string,
 	log.Functionf("handleContentTreeDelete(%s)", key)
 	config := configArg.(types.ContentTreeConfig)
 	ctx := ctxArg.(*volumemgrContext)
-	ok := handleContentTreeClusterForUs(config)
-	if !ok {
+	// Do nothing if it is non-local
+	if !config.IsLocal {
```

Comment:
the ContentTreeStatus created by volumemgr does not need to be updated and deleted?

Reply:
It will be deleted when app is deleted I think. I can test it out